src/resultsdisassemblypage.cpp

```diff
@@ -199,3 +201,46 @@ void ResultsDisassemblyPage::setData(const Data::DisassemblyResult &data)
         m_objdump = QLatin1String("aarch64-linux-gnu-objdump");
     }
 }
+
+void ResultsDisassemblyPage::resetCallStack()
+{
+    m_callStack.clear();
+}
+
+void calcFunction(QString asmLineCall, QString* offset, QString* symName)
+{
+    QStringList sym = asmLineCall.trimmed().split(QLatin1Char('<'));
```

Comment:
please document an expected string line and the expected parsed output

Reply:
remove the trimmed() (not needed as you can trim the results below anyways), use splitRef